packages/core-sdk/test/integration/dispute.test.ts

```diff
@@ -16,59 +23,101 @@ describe.skip("Dispute Functions", () => {
     senderAddress = config.account.address;
     client = StoryClient.newClient(config);
   });
-  /*
-  it("should be able to whitelist dispute tags and wait for transaction", async () => {
-    const response = await expect(
-      client.dispute.whitelistDisputeTags({
-        tag: "testTag",
-        allowed: true,
+  describe("[Write Functions] Should be able to", async function () {
```

Comment:
All write functions are skipped right now. Are they working?

Reply:
Doesn't work unless we have a working ArbitrationPolicy fixture

Reply:
Also, setDisputeJudgement won't work unless we set up the whitelisted Arbitration relayer contract (or whatever it's called)